src/app/core/http-content-type-header-interceptor/http-content-type-header-interceptor.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpHandler, HttpInterceptor, HttpRequest } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+
+@Injectable()
+export class HttpContentTypeHeaderInterceptor implements HttpInterceptor {
+    construct () {
+
+    }
+
+    intercept(req:HttpRequest<any>, next: HttpHandler) {
+        var method = req.method
+        var urlWithParams = req.urlWithParams
+        // If the request contains a content type, be sure to set the encoding to utf-8        
+        if(['POST', 'PUT'].includes(method)) {
+            var clonedRequest;
+            if(urlWithParams == '/signin/auth.json') {
```

Comment:
this header excepcion seems to be require for `userStatus.json?logUserOut=true, sendReactivation.json, social/signin/auth.json, shibboleth/signin/auth.json, signin/auth.json`


Maybe we can declare a class level atribute like

```
const formUrlEcondedUrls = ["userStatus.json?logUserOut=true", "sendReactivation.json", "social/signin/auth.json", "shibboleth/signin/auth.json", "signin/auth.json"];
```

And check if a route is in the array 
``` 
if(formUrlEcondedUrls.includes(urlWithParams)) {
```